mcq/tasks/request_decrease_item_quantity.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import logging
+from constants.service_constants import USER_SERVICE_DECREASE_ITEM_QUANTITY_PATH
+
+logger = logging.getLogger(__name__)
+
+
+def request_decrease_item_quantity(user_email: str, item_type: str):
+    if not user_email:
+        logger.error("Error while decreasing item quantity for user. User email is required")
+        return False, "User email is required"
+
+    if not item_type:
+        logger.error("Error while decreasing item quantity for user {}. Item type is required".format(user_email))
```

Comment:
In my personal opinion saying something like this is more direct:
"Failed to decrease item quantity for user"

Because the logs already know that this log is an error log when we say `logger.error`